lib/jsdom/living/helpers/shadow-dom.js

```diff
@@ -210,6 +210,30 @@ function findSlot(slotable, openFlag) {
   return null;
 }
 
+// https://dom.spec.whatwg.org/#concept-shadow-including-descendant
+function* shadowDomIncludingInclusiveDescendantIterator(node) {
```

Comment:
Can we name these "shadowIncludingInclusiveDescendantsIterator" / "shadowIncludingDescendantsIterator"? That seems to fit the spec better.